scripts/update/04-chmod.sh

```diff
@@ -3,7 +3,8 @@
 
 echo_progress_start "Checking permissions on swizzin files"
 
-chmod -R 700 /etc/swizzin/scripts # Set permissions of all scripts
-chmod -R 600 /etc/swizzin/sources # Sources are only `.` or `source`d and does not need to be executable
+chmod -R 700 /etc/swizzin/scripts        # Set permissions of all scripts
+chmod -R 600 /etc/swizzin/sources        # Sources are only `.` or `source`d and does not need to be executable
+chmod 700 /etc/swizzin/sources/functions # Best to keep X permission on the folder inside sources
```

Comment:
Uhhh this should work, no? If I can make sure I can forget about this file and not keep the hardcoded dirs up to date I won't mind
```suggestion
find /etc/swizzin/sources -type d -exec chmod 700 {} \; # Best to keep X permission on the folder inside sources
```

Reply:
Aaaah of course! that's great.